inc/functions_task.php

```diff
@@ -68,6 +70,15 @@ function run_task($tid=0)
 		);
 		$db->update_query("tasks", $updated_task, "tid='{$task['tid']}'");
 
+		if(!empty($task['file']))
```

Comment:
Foremost, this should be outside the if statement.

Regardless, seems like this file needs no edition, as the task title for logging seems to be managed in the following block:

https://github.com/mybb/mybb/blob/7e0247a0874cc28c5b5c44c98d18139738f2140f/admin/modules/tools/tasks.php#L677-L694

As you can see, the code fetches the task name directly from the database (it even ignores the possibility of the task no longer existing...).

Regards.

Reply:
I will add the translation to this for the tasks logs which I forgot, but the change in `functions_task` was actually to translate the name of the task being sent by email when a task fail.